src/EditorFeatures/CSharpTest2/Recommendations/NativeIntegerKeywordRecommenderTests.cs

```diff
@@ -284,5 +284,11 @@ class C
 {
     delegate*$$");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestInPreprecessorDirective()
+        {
+            await VerifyAbsenceAsync("#$$");
```

Comment:
Does this test fail before making the fix?
I'm not sure whether this is tested against both regular & scripting modes or not.

Reply:
Yes, it fails without the fix. And yes, VerifyAbsendeAsync ultimately tests regular and script modes (in `RecommenderTests.VerifyAbsenceAsync`)